lib/Dialect/FIRRTL/Transforms/LayerSink.cpp

```diff
@@ -138,9 +138,9 @@ class EffectInfo {
     // Regular modules may be pure.
     if (auto m = dyn_cast<FModuleOp>(op))
       return update(m);
-    // Memory modules are pure.
-    if (auto m = dyn_cast<FMemModuleOp>(op))
-      return;
+//    // Memory modules are pure.
+//    if (auto m = dyn_cast<FMemModuleOp>(op))
+//      return;
```

Comment:
High level: commented out code should never be committed.

Reply:
Thanks, updated to a newer patch :)